src/backend/app/auth/auth_routes.py

```diff
@@ -248,33 +270,16 @@ async def temp_login(
     Returns:
         Response: The response object containing the access token as a cookie.
     """
-    access_token = settings.OSM_SVC_ACCOUNT_TOKEN
-
-    if not access_token:
-        raise HTTPException(
-            status_code=HTTPStatus.FORBIDDEN,
-            detail=(
-                "OSM_SVC_ACCOUNT_TOKEN variable is not set. Temp login not possible."
-            ),
-        )
-
-    response = Response(status_code=HTTPStatus.OK)
-    cookie_name = settings.FMTM_DOMAIN.replace(".", "_")
-    log.debug(
-        f"Setting TEMP cookie in response named '{cookie_name}' with params: "
-        f"max_age=604800 | expires=604800 | path='/' | "
-        f"domain={settings.FMTM_DOMAIN} | httponly=True | samesite='lax' | "
-        f"secure={False if settings.DEBUG else True}"
-    )
-    response.set_cookie(
-        key=cookie_name,
-        value=access_token,
-        max_age=604800,
-        expires=604800,  # expiry set to 7 days,
-        path="/",
-        domain=settings.FMTM_DOMAIN,
-        secure=False if settings.DEBUG else True,
-        httponly=True,
-        samesite="lax",
-    )
-    return response
+    username = "svcfmtm"
```

Comment:
We can probably remove reference to svcfmtm in this token, as it's no longer an osm account, just a temp jwt

Reply:
We talked about this previously in the above comment to hard-code the username to `svcfmtm`. If not, we need to create a temporary user.

Reply:
Oh of course, my bad!
I shouldn't do a review after 3 hours sleep 😂